client/src/components/user-admin/UserManagement.js

```diff
@@ -1,6 +1,13 @@
 import React, { useState } from 'react';
+import {Box, Button, ButtonGroup, TextField, Typography, List, ListItem, ListItemButton} from '@mui/material';
+
+
 import '../../sass/UserAdmin.scss';
 
+const Buttonsx = {
+  px: "1rem",
```

Comment:
Please switch to numberical values for margin, padding, etc.

Details on MUI spacing system [here](https://mui.com/system/spacing/).

If we're using numerical values we can go and chagne the theme later on and the spacing of the app will automatically update. This won't happen if we've got hardcoded strings.

Reply:
Looking at this closer, if this prop is only going to be 1 or two values, we might as well just define it inline with the Box it's a part of.

We also might need to limit the top and bottom padding with `py` to fit the design.

Reply:
the object is referring to the `2 buttons` inside the `buttongroup` moving those property inline would increase the redundancy and will have to update it in two places instead of one.

Reply:
Gotcha, my bad